### PR TITLE
Wait for button release for exiting

### PR DESCRIPTION
**Pull request by eried**
Solves https://github.com/DeviationTX/deviation/issues/995

### CODE REVIEW

File: src/pages/128x64x1/advanced/mixer_curves.c [hunk 1]
```diff
@@ -95,9 +95,10 @@ void PAGE_EditCurvesInit(int page)
 static unsigned action_cb(u32 button, unsigned flags, void *data)
 {
     (void)data;
-    if ((flags & BUTTON_PRESS) || (flags & BUTTON_LONGPRESS)) {
+    if ((flags & BUTTON_PRESS) || (flags & BUTTON_LONGPRESS) || (flags & BUTTON_RELEASE)) {
         if (CHAN_ButtonIsPressed(button, BUT_EXIT)) {
-            PAGE_Pop();
+            if (flags & BUTTON_RELEASE)
```
Comment (PhracturedBlue) [inline src/pages/128x64x1/advanced/mixer_curves.c:100]: Lint Error:
```
100:  Line ends in whitespace.  Consider deleting these extra spaces.  
```

Comment (goebish) [inline src/pages/128x64x1/advanced/mixer_curves.c:100]: please remove the extra space at the end of the line so the linter stops complaining

Comment (eried) [reply src/pages/128x64x1/advanced/mixer_curves.c]: OK, strange that I cannot see the linter errors 🤔

Comment (TheRealMoeder) [reply src/pages/128x64x1/advanced/mixer_curves.c]: it's down in the bottom at the GitHub checks or just run make lint

**Pull request merged**